lang/english.xml

```diff
@@ -1598,10 +1583,50 @@
+		</Replace>
+		<Replace Tag="LOC_LEADER_TRAIT_PRESLAV_DESCRIPTION" Language="en_US">
+			<Text>You receive +40 Loyalty per turn in all cities.</Text>
+		</Replace>
+		<Replace Tag="LOC_IMPROVEMENT_MEKEWAP_DESCRIPTION" Language="en_US">
+			<Text>Unlocks the Builder ability to construct a Mekewap, unique to Cree.[NEWLINE][NEWLINE]Provides +1 [ICON_PRODUCTION] Production and +1 [ICON_Housing] Housing. +1 [ICON_GOLD] Gold if adjacent to a Luxury resource. +1 [ICON_FOOD] Food for every 2 adjacent Bonus Resources. Additional +1 [ICON_PRODUCTION] Production and +1 [ICON_Housing] Housing with Civil Service civic. Additional +2 [ICON_GOLD] gold for each adjacent Luxury Resource with Cartography technology. Must be placed adjacent to a Bonus or Luxury resource. Cannot be built adjacent to another Mekewap.</Text>
+		</Replace>
+		<Replace Tag="LOC_BUILDING_GOV_SCIENCE_DESCRIPTION" Language="en_US">
+			<Text>Builders and Military Engineers gain the ability to use all of their charges to provide bonus [ICON_Production] Production to a District Project. Each charge grants 2% of project [ICON_Production] Production. Once per city per turn.[NEWLINE]Awards +1 [Icon_Governor] Governor Title.</Text>
+		</Replace>
+		<Replace Tag="LOC_GREAT_PERSON_GRANT_LOTSO_SCIENCE" Language="en_US">
+			<Text>Grants 1000 [ICON_SCIENCE] Science and [ICON_CULTURE] Culture (on Standard speed). Can only be activated on Campus in other civilization.</Text>
+		</Replace>
+		<Replace Tag="LOC_UNIT_ANTIAIR_GUN_DESCRIPTION" Language="en_US">
+			<Text>Atomic era anti-air support unit. Provides cover from air attacks up to 1 hex away from the weapon. +5 [ICON_Strength] Combat Strength for each adjacent anti-air unit as Support Bonus.</Text>
+		</Replace>
+		<Replace Tag="LOC_UNIT_MOBILE_SAM_DESCRIPTION" Language="en_US">
+			<Text>Information era anti-air support unit. Provides cover from air and nuclear attacks up to 1 hex away from the weapon. +5 [ICON_Strength] Combat Strength for each adjacent anti-air unit as Support Bonus.</Text>
+		</Replace>
+		<Replace Tag="LOC_UNIT_BIPLANE_DESCRIPTION" Language="en_US">
+			<Text>First air combat unit, available in the Modern era. +5 [ICON_Strength] Combat Strength for each adjacent air unit as Support Bonus.</Text>
+		</Replace>
+		<Replace Tag="LOC_UNIT_FIGHTER_DESCRIPTION" Language="en_US">
+			<Text>Atomic era Biplane upgrade. +5 [ICON_Strength] Combat Strength for each adjacent air unit as Support Bonus.</Text>
+		</Replace>
+		<Replace Tag="LOC_UNIT_JET_FIGHTER_DESCRIPTION" Language="en_US">
+			<Text>Information era Fighter upgrade. +5 [ICON_Strength] Combat Strength for each adjacent air unit as Support Bonus.</Text>
+		</Replace>
```

Comment:
I don't know if adding this information is the good way to document how aerial support bonus work.
Documenting this on the military tradition civic, with other unit support/flank bonus seem more better for me.

Reply:
PS: I agree this should be documented in the game if it's not, but putting this under units feels more like "Unique bonus", but it actually work with all aerial unit.

Reply:
I know Civpedia have "Game Mecanics" entry or something like this, but it's not very sure about it.
You could also take advantage of it to write here that "ranged unit don't recieve the support bonus".